docs/csharp/language-reference/keywords/for.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "for (C# Reference)"
+title: C# for statement
```

Comment:
I changed a few titles to improve SEO. Let me know if you don't want me to change these though.

Reply:
I like this.  I agree it will be more findable.